src/Google.Api.Gax/AssemblyInfo.cs

```diff
@@ -10,3 +10,5 @@
 [assembly:InternalsVisibleTo("Google.Api.Gax.Testing,PublicKey=0024000004800000940000000602000000240000525341310004000001000100bfa2d20e82583e5f3a47efa518d8a7cda58d6e1b61f51ed3de3900f238fb01e0ca95b3ed1c68147e4865a336b252f070d5207e653c9014d271fc829c18981d1c80d418d300545c3ad0e1ff4d77c596177dbfe0a588fff57306f1d6ba655de3cfa520ad777ea73bb7a92fd6027071c94bb12d5871fcd076299de0502bca12e3aa")]
 [assembly:InternalsVisibleTo("Google.Api.Gax.Tests,PublicKey=0024000004800000940000000602000000240000525341310004000001000100bfa2d20e82583e5f3a47efa518d8a7cda58d6e1b61f51ed3de3900f238fb01e0ca95b3ed1c68147e4865a336b252f070d5207e653c9014d271fc829c18981d1c80d418d300545c3ad0e1ff4d77c596177dbfe0a588fff57306f1d6ba655de3cfa520ad777ea73bb7a92fd6027071c94bb12d5871fcd076299de0502bca12e3aa")]
 [assembly:InternalsVisibleTo("Google.Api.Gax.Grpc,PublicKey=0024000004800000940000000602000000240000525341310004000001000100bfa2d20e82583e5f3a47efa518d8a7cda58d6e1b61f51ed3de3900f238fb01e0ca95b3ed1c68147e4865a336b252f070d5207e653c9014d271fc829c18981d1c80d418d300545c3ad0e1ff4d77c596177dbfe0a588fff57306f1d6ba655de3cfa520ad777ea73bb7a92fd6027071c94bb12d5871fcd076299de0502bca12e3aa")]
+[assembly: InternalsVisibleTo("Google.Api.Gax.Rest,PublicKey=0024000004800000940000000602000000240000525341310004000001000100bfa2d20e82583e5f3a47efa518d8a7cda58d6e1b61f51ed3de3900f238fb01e0ca95b3ed1c68147e4865a336b252f070d5207e653c9014d271fc829c18981d1c80d418d300545c3ad0e1ff4d77c596177dbfe0a588fff57306f1d6ba655de3cfa520ad777ea73bb7a92fd6027071c94bb12d5871fcd076299de0502bca12e3aa")]
```

Comment:
We want to be able to get at this from Google.Api.Gax.Rest, but not from other code. Note how the previous line is similar for Grpc, although I'm not sure which internal type that's related to.